torch/_inductor/codecache.py

```diff
@@ -2143,9 +2143,15 @@ def _pad_to_alignment(raw_bytes: bytes) -> bytes:
                         # Include SASS for the current specific arch
                         f"-gencode arch=compute_{current_arch},code=sm_{current_arch} "
                     )
-                    subprocess.run(
-                        cmd.split(), capture_output=True, text=True, check=True
-                    )
+                    try:
+                        subprocess.run(
+                            cmd.split(),
+                            capture_output=True,
+                            text=True,
+                            check=True,
+                        )
+                    except subprocess.CalledProcessError as e:
+                        print(f"{cmd} failed with: {e.stderr}")
```

Comment:
```suggestion
                        print(f"{cmd} failed with:\nstdout:\n{e.stdout}\nstderr:\n{e.stderr}", file=sys.stderr)
                        raise
```

I assume you want to re-raise here?